### PR TITLE
JENKINS-67522 Use same animation as core for pipeline logs

### PR DESCRIPTION
**Pull request by timja**
see https://issues.jenkins.io/browse/JENKINS-67552

Introduced in 2.320 in https://github.com/jenkinsci/jenkins/pull/5871

We could do version detection but up to maintainers, this would be the simplest approach

### CODE REVIEW

**Review by jglick — APPROVED** [review]
OK? There are also two spots in `workflow-support-plugin` that would need the same treatment.

**Review by car-roll — APPROVED** [review]

**Comment by timja** [comment]
> OK? There are also two spots in `workflow-support-plugin` that would need the same treatment.

There’s quite a few references in plugins but this is the most visible one, others can prob wait a bit longer I think

**Comment by timja** [comment]
FML:
https://github.com/jenkinsci/bom/blob/231c8d6403cf235a1f149fd24db857395d510cad/excludes.txt#L16-L17

File: src/test/java/org/jenkinsci/plugins/workflow/job/WorkflowRunRestartTest.java [hunk 1]
```diff
@@ -270,7 +270,7 @@
 
     private boolean hasTermOrKillLink(WorkflowRun b, String termOrKill) throws Exception {
         return !story.j.createWebClient().getPage(b)
-                .getByXPath("//a[@href = '#' and contains(@onclick, '/" + b.getUrl() + termOrKill + "')]").isEmpty();
+                .getByXPath("//a[@href = '#' and contains(@data-url, '/" + b.getUrl() + termOrKill + "')]").isEmpty();
```
Comment (timja) [inline src/test/java/org/jenkinsci/plugins/workflow/job/WorkflowRunRestartTest.java:273]: Behaviour changed in https://github.com/jenkinsci/jenkins/pull/5515 in 2.320

Not picked up in bom due to https://github.com/jenkinsci/bom/blob/231c8d6403cf235a1f149fd24db857395d510cad/excludes.txt#L16-L17

**Comment by MarkEWaite** [comment]
@timja the title of the pull request includes the wrong issue ID.  You reference the correct issue ID as https://issues.jenkins.io/browse/JENKINS-67552

Could the changelog be corrected to link to the correct issue ID https://issues.jenkins.io/browse/JENKINS-67552?

JENKINS-67522 is the incorrect issue ID

**Comment by timja** [comment]
Sorted thanks

**Pull request merged**